test/collection.json

```diff
@@ -626,13 +616,60 @@
 				}
 			},
 			"response": []
+		},
+		{
```

Comment:
For some reason, I receive an error from the postman, when I try to open this new collection. 
![image](https://user-images.githubusercontent.com/48105857/227012887-ae50f436-76de-4179-a678-2bbbeb8e44db.png)